runbotics/runbotics-desktop/src/core/bpm/runtime/runtime.service.ts

```diff
@@ -580,8 +582,15 @@ export class RuntimeService implements OnApplicationBootstrap, OnModuleDestroy {
         definition.environment.assignVariables(globalVariables);
     }
 
+    public isAnySequencesWithoutExpression = (owner: IActivityOwner) => owner.outbound
+        .find(outbound => this.isSequenceWithoutExpression(outbound));
+
+    private isSequenceWithoutExpression = (outbound: IOutboundSequence): boolean => !outbound?.isDefault &&
+        (outbound?.behaviour?.conditionExpression?.body === null || 
+            outbound?.behaviour?.conditionExpression?.body === undefined);
```

Comment:
```suggestion
        Boolean(outbound?.behaviour?.conditionExpression?.body);
```

Reply:
I've changed it to !outbound?.behaviour?.conditionExpression?.body, because it maps to Boolean too.